GNUmakefile

```diff
@@ -0,0 +1,32 @@
+MKDIR_P := mkdir -p
+RM_F := rm -rf
+
+export GO111MODULE=on
+
+Targets := \
+	echoserver \
+	ingress-controller-conformance
+
+build: $(Targets) ## Build the conformance tool
+
+.PHONY: echoserver
+echoserver:
+	go build -o $@ tools/echoserver.go
+
+.PHONY: ingress-controller-conformance
+ingress-controller-conformance: internal/pkg/assets/assets.go
+	go build -o $@ .
```

Comment:
we should probably build with a go image for reproducibility.

Reply:
For all builds or for CI and release builds?